### DISCUSSION TITLE
Upgrade your version of pocketbase in local server and live on fly.io app

### DISCUSSION THREAD

**Discussion by TigerJohn3** [pocketbase/pocketbase#2615] [Q&A] [answered]
Hi all,

I apologize if this is a dumb question and I haven't done enough digging to find the answer...

But, how do you upgrade your version of pocketbase, both on the local development server, and on my live fly.io server?

I found one other discussion post on this, but it didn't make sense for me.

Is there a command that can be ran to upgrade everything without losing data?

Thank you in advance.

John

**Comment by ganigeorgiev** [comment, marked as answer]
If you use the prebuilt executable and followed the guide from https://github.com/pocketbase/pocketbase/discussions/537, you can change the version from the `PB_VERSION` arg in the Dockerfile (OR by updating it in the `[[build.args]]` section in your fly.toml config) and then redeploy.

Locally, if you are not using Docker, you can download the latest version and replace your current binary.
With v0.16.0 there is a helper `./pocketbase update` command that will download the latest executable, backup your `pb_data`, and replace your current executable with the previously downloaded one (the command currently doesn't work on Windows, but a fix has been applied already in `master` and will be available with the next v0.16.4 release this week).

If you are using PocketBase as framework, then to update your PocketBase version you can update your go.mod file, like any other Go package.

**Reply by TigerJohn3** [reply]
Thank you kind sir. This worked perfectly. I have never used fly.io before setting up this pocketbase project I'm working on. 

Thank you for all the work with this and for staying on top of all the questions/discussions in this forum.
